src/interfaces/audit.interface.ts

```
@@ -0,0 +1,25 @@
+import { AuditLogActionsEnum } from "../enums/AuditLogAction";
+
+export interface CreateAditLogParams {
+  entityType: string;
+  entityId: string;
+  action: AuditLogActionsEnum;
+  oldValues?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)